### PR TITLE
Nymph and Scannerbed fixes

### PR DESCRIPTION
**Pull request by NanakoAC**
Fixes an unreported issue with carbon attack procs that made diona nymphs (and i guess xenomorphs and slimes too) immune to melee attacks.

Nymphs no longer gain light energy while ventcrawling. The inside of a pipe is treated as total darkness.
Nymph max health reduced by 15%. I don't want to nerf them too much.

Fixes several runtime errors in life procs, where it tried to set client vars without checking for a client. I ran into them when shooting a nymph

Fixes the body scanner in medical:
Bay has switched from a static single-direction sprite, to a quad-directional scanner. Though i couldn't find any way to rotate it ingame.  I had to edit the instance to change its direction, so thats the map edit.

They also screwed up the sprites a little bit. The gentle pulsing animated light was gone, so i valiantly opened photoshop and remade it.

### CODE REVIEW

File: code/modules/mob/living/carbon/carbon_defense.dm [hunk 1]
```diff
@@ -1,9 +1,9 @@
 
 //Called when the mob is hit with an item in combat.
-/mob/living/carbon/resolve_item_attack(obj/item/I, mob/living/user, var/effective_force, var/hit_zone)
```
Comment (NanakoAC) [inline code/modules/mob/living/carbon/carbon_defense.dm]: This effective force var seems to have been erroneously copied from the proc below it, and doesn't belong here.

File: maps/exodus-1.dmm [hunk 2]
```diff
@@ -5254,7 +5254,7 @@
 "bXb" = (/obj/machinery/atmospherics/unary/vent_scrubber/on{dir = 1},/obj/effect/floor_decal/corner/red{tag = "icon-corner_white (SOUTHWEST)"; icon_state = "corner_white"; dir = 10},/turf/simulated/floor/tiled,/area/medical/sleeper)
 "bXc" = (/obj/machinery/alarm{dir = 4; icon_state = "alarm0"; pixel_x = -22},/obj/machinery/light{icon_state = "tube1"; dir = 8},/obj/machinery/atmospherics/pipe/simple/visible,/turf/simulated/floor/tiled,/area/medical/sleeper)
 "bXd" = (/obj/machinery/body_scanconsole{tag = "icon-body_scannerconsole (WEST)"; icon_state = "body_scannerconsole"; dir = 8},/obj/effect/floor_decal/corner/red{tag = "icon-corner_white (SOUTHWEST)"; icon_state = "corner_white"; dir = 10},/turf/simulated/floor/tiled,/area/medical/sleeper)
-"bXe" = (/obj/machinery/bodyscanner{tag = "icon-body_scanner_0 (EAST)"; icon_state = "body_scanner_0"; dir = 4},/obj/effect/floor_decal/corner/red{tag = "icon-corner_white (SOUTHWEST)"; icon_state = "corner_white"; dir = 10},/turf/simulated/floor/tiled,/area/medical/sleeper)
```
Comment (NanakoAC) [inline maps/exodus-1.dmm]: I think this is the smallest map change i've ever done

File: code/modules/mob/living/life.dm [hunk 3]
```diff
@@ -135,7 +135,8 @@
 	return 1
 
 /mob/living/proc/handle_vision()
-	client.screen.Remove(global_hud.blurry, global_hud.druggy, global_hud.vimpaired, global_hud.darkMask, global_hud.nvg, global_hud.thermal, global_hud.meson, global_hud.science)
+	if (client)
```
Comment (skull132) [inline code/modules/mob/living/life.dm]: This should be unnecessary.

The code right above should stop handle_vision from being called if there is no client in the body. I know that carbon/human/life() obeys by this.

If carbon/ayylmao/life() does not, then you should look into fixing some overrides you've got going on there. It should be unnecessary to perform this many sanity checks here.

File: maps/exodus-1.dmm (hunk 2 above)
Comment (Alberyk) [inline maps/exodus-1.dmm]: You are probably missing the scanners on central command and the mercenary ship.

Comment (NanakoAC) [inline maps/exodus-1.dmm]: yep i missed threee on other zlevels
fixing

**Comment by NanakoAC** [comment]
changes made as requested

**Pull request merged**